demisto_sdk/common/hook_validations/release_notes.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import print_function
+
+import os
+import re
+from demisto_sdk.common.tools import run_command
+
+from demisto_sdk.common.tools import print_error, get_latest_release_notes_text, \
+    get_release_notes_file_path
+
+release_notes_standard = 'https://github.com/demisto/content/blob/master/docs/release_notes/README.md'
+
+
+class ReleaseNotesValidator:
+    """Release notes validator is designed to ensure the existence and correctness of the release notes in content repo.
+
+    Attributes:
+        file_path (str): the path to the file we are examining at the moment.
+
+    """
+
+    def __init__(self, file_path):
+        # type: (str) -> None
+        self.file_path = file_path
+        self.release_notes_path = get_release_notes_file_path(self.file_path)
```

Comment:
i think that maybe this function should be here, where is it being used?

Reply:
It's also being used in the `base_validator`. I think it makes sense for it to be in the common tools.